docs/visualization/lupton_rgb.rst

```diff
@@ -0,0 +1,29 @@
+**********************************
```

Comment:
Got it, thanks. RST in general is not as picky so long as the header are at least as long as the text.

That said, the docs build is still failing on my local machine, with the same error.